packages/types/src/inventory/common.ts

```diff
@@ -267,16 +350,42 @@ export type CreateInventoryLevelInput = {
   incoming_quantity?: number
 }
 
+/**
+ * @interface
+ * 
+ * The attributes to update in an inventory level.
+ * 
+ * @prop stocked_quantity - The stocked quantity of the associated inventory item in the associated location.
+ * @prop incoming_quantity - The incoming quantity of the associated inventory item in the associated location.
+ */
 export type UpdateInventoryLevelInput = {
   stocked_quantity?: number
   incoming_quantity?: number
 }
 
+/**
+ * @interface
+ * 
+ * The attributes to update in an inventory level. The inventory level is identified by the IDs of its associated inventory item and location.
+ * 
+ * @prop inventory_item_id - The ID of the associated inventory level.
+ * @prop location_id - The ID of the associated location.
```

Comment:
question: should we include `stocked_quantity` and `incoming_quantity` in this?

Reply:
It includes the TSDoc comments for them defined in `UpdateInventoryLevelInput` by default.